app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -531,6 +531,15 @@ BrowserWorld::State::UpdateControllers(bool& aRelayoutWidgets) {
     if (controller.modelToggle)
       controller.modelToggle->ToggleAll(controller.mode == ControllerMode::Device);
 
+    if (controller.leftHandActionEnabled && controller.handActionButtonTransform != nullptr) {
+      vrb::Matrix matrix = controller.handJointTransforms[10]->GetTransform();
```

Comment:
Accessing index 10 is confusing. We only have a OpenXR hand tracking implementation so I guess you're accessing the location of XR_HAND_JOINT_INDEX_TIP_EXT ?

In any case we should make it independent of the SDK. Maybe we could define an enum with the 26 hand joints from the OpenXR spec and hope for the best. Another option would be to ask the device delegate maybe...